App.Loans/Repositories/LoansRepository.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using App.Loans.Models;
+using App.Configuration;
+using App.Repositories;
+
+namespace App.Loans.Repositories
+{
+    public interface ILoansRepository
+    {
+        IEnumerable<Loan> GetAll();
+        IEnumerable<string> GetValues();
```

Comment:
What this interface method did ? Is it really needed, because you already have GetAll method ?

Reply:
Yes i need it for resolve the task.

Reply:
Not a good explanation